sass/style-eu.scss

```diff
@@ -7,13 +7,9 @@
 //@use '@ecl/eu-component-form-help-block/eu-component-form-help-block';
 @use '@ecl/theme-dev/maps/color';
 @use "./compositions/content_item";
-@use "./compositions/content_language_switcher" with (
-  $_core_color: map.get(theme.$color, 'blue-100'),
-);
```

Comment:
Why removing the language selector?

Reply:
Styles from this files aren't used anymore. Please check https://github.com/openeuropa/oe_theme/pull/1105/files#diff-2ff0353a221baabf5fe9662567a9de59fc9b3304137bb5b219a0cbd7ef25fbf3.
Language switcher isn't included in the page header in ECL v.3